# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/diligent-engine/no-clang-format.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/DiligentTools/RenderStateNotation/CMakeLists.txt b/DiligentTools/RenderStateNotation/CMakeLists.txt
index 429f29c..9ffa3b5 100644
--- a/DiligentTools/RenderStateNotation/CMakeLists.txt
+++ b/DiligentTools/RenderStateNotation/CMakeLists.txt
@@ -45,11 +45,7 @@ source_group("generated" FILES ${GENERATED})
 source_group("source"    FILES ${SOURCE})
 source_group("scripts"   FILES ${SCRIPTS})
 
-if (EXISTS "${CLANG_FORMAT_EXECUTABLE}")
-    set(FORMATTING_COMMAND "${CLANG_FORMAT_EXECUTABLE}" -i *.hpp --verbose)
-else()
-    set(FORMATTING_COMMAND "${CMAKE_COMMAND}" -E echo "clang-format executable is not found: RSN parser headers will not be formatted")
-endif()
+set(FORMATTING_COMMAND "${CMAKE_COMMAND}" -E echo "clang-format executable is not found: RSN parser headers will not be formatted")
 
 add_custom_command(OUTPUT ${GENERATED}
                    COMMAND ${Python3_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/scripts/cxx_generator.py" --dir "." --files ${REFLECTED}
